ci-operator/step-registry/openshift/e2e/vsphere/capi/openshift-e2e-vsphere-capi-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-e2e-vsphere-capi
+  steps:
+    pre:
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-vsphere
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-core-dump
+    - chain: ipi-vsphere-post
+  documentation: |-
+    The Openshift E2E vSphere workflow for test Cluster API integration.
```

Comment:
This is now identical to `openshift-e2e-vsphere` right? So it isn't needed?

Reply:
The `steps.pre` is different for all kinds of providers:
Capi version has:
```
    pre:
    - ref: ipi-conf
    - ref: ipi-conf-telemetry
    - ref: ipi-conf-vsphere
    - chain: ipi-install
```

Not capi version:
```
pre:
    - chain: ipi-vsphere-pre
```

And `ipi-vsphere-pre`:

```
chain:
  as: ipi-vsphere-pre
  steps:
  - chain: ipi-conf-vsphere
  - chain: ipi-install-vsphere
  documentation: |-
    The IPI setup step contains all steps that provision an OpenShift cluster
    with a default configuration on vSphere.
```